packages/tradershub/src/features/cfd/screens/TradeScreen/TradeScreen.tsx

```diff
@@ -86,26 +86,26 @@ const MT5TradeScreen = ({ account }: MT5TradeScreenProps) => {
                 <div className='flex flex-col gap-100'>
                     {platform === mt5Platform && (
                         <Fragment>
-                            <MT5TradeDetailsItem label='Broker' value='Deriv Holdings (Guernsey) Ltd' />
-                            <MT5TradeDetailsItem
+                            <TradeDetailsItem label='Broker' value='Deriv Holdings (Guernsey) Ltd' />
```

Comment:
Value is wrong
Its `Deriv.com Limited` for now
Better to add a getBrokerName function as we are going to have this value from backend soon.

Reply:
@hamza-deriv Can we get this values from BE now? If no, then we can do it later. Also, in my opinion, it's out of the scope of this card. I've just renamed the components in this PR

Reply:
Okay sure